goldmane/pkg/emitter/emitter.go

```diff
@@ -92,7 +92,7 @@ func NewEmitter(opts ...Option) *Emitter {
 	return e
 }
 
-func (e *Emitter) Run(stopCh chan struct{}) {
+func (e *Emitter) Run(ctx context.Context) {
```

Comment:
hmmm, this isn't used - probably best to at least add a TODO if we're not going to check the context in the loop (my preference)

Reply:
Ya I can add that, but I was really just replacing what was currently there (which wasn't used).

Reply:
Yeah I understand, but the change is pointless if it doesn't do anything. You can do it now (in which case we remember) or you can do it later as part of your FV PR, in which case we might forget and then we'll see odd flaky hard to trace behavior in the FVs.

Reply:
Added a commit to handle the context.